docker/app.conf

```diff
@@ -0,0 +1,9 @@
+<VirtualHost *:80>
+    ServerName localhost
+    DocumentRoot /var/www/app/public
+    <Directory /var/www/app/public>
+        Options -Indexes
+        AllowOverride All
+        Require all granted
+    </Directory>
+</VirtualHost>
```

Comment:
Why do you have virtual host if you are using nginx? Isn't this only for Apache?

Reply:
Yes, this is wrong, I'll gonna remove it and commit again.